Gemfile

```diff
@@ -61,8 +54,6 @@ gem 'daemons'
 gem 'delayed_job_active_record'
 gem 'delayed_job_web'
 
-# Spree's default pagination gem (locked to the current version used by Spree)
-# We use it's methods in OFN code as well, so this is a direct dependency
```

Comment:
We still use this in OFN code, right? The comment is valid.

Reply:
as far as I understand the comment is valid in so far as it's a direct dependency of OFN.
But because we dont have spree anymore I dont think we need to mention history and most other dependencies are also direct dependencies, so I thought the comment was not needed now. It's just a good/normal dependency.